src/components/OcCheckbox.vue

```diff
@@ -1,17 +1,17 @@
 <template>
-  <label :for="id" :class="{ 'oc-cursor-pointer': !disabled }">
+  <span :class="{ 'oc-cursor-pointer': !disabled }">
```

Comment:
As with the radiobutton, please move the `oc-cursor-pointer` class to the label and all the conditionally applied classes of the label into a computed property.